app/src/org/commcare/activities/connect/ConnectManager.java

```diff
@@ -588,12 +588,10 @@ public static void forgetAppCredentials(String appId, String userId) {
     }
 
     public static AuthInfo.ProvidedAuth getCredentialsForApp(String appId, String userId) {
```

Comment:
add `@Nullable` to signify possible null return

Reply:
[49aef9b](https://github.com/dimagi/commcare-android/pull/2724/commits/49aef9b1630835182867aa080277d058dbd245c5)